package.json

```diff
@@ -26,6 +26,13 @@
   "homepage": "https://github.com/Shopify/shopify-node-api",
   "author": "Shopify Inc.",
   "license": "MIT",
+  "keywords": [
+    "shopify",
+    "node",
+    "jwt",
+    "typescript",
+    "javascript"
```

Comment:
I'm wondering about the keywords a bit - does it make sense to mention typescript / javascript / node?

I'd consider adding something like `shopify, app, graphql, rest, webhook` - does that make sense?

Reply:
I suppose it doesn't make sense for `typescript`, `javascript` or `node`, since it's hosted on `npmjs.org` anyway!  Maybe the `node` one to distinguish it from something that works in browsers?

The others make sense, will update.

Reply:
Good point about `node`, let's keep that!

Reply:
Would "Shopify API Library" make sense in this list as well? Sort of how `themekit` has the keyword "themekit" as a direct reference to itself?

Reply:
Also, FWIW, the existing 3rd party libraries for node seem to mostly have just "Shopify" and "API" as their keywords 🤷 Maybe less is more?